tests/annotator/annotator.cpp

```diff
@@ -1513,10 +1556,10 @@ TEST(Annotator, automaticIdAllItemsEntityType)
         model->component("component2")->variable("variable2")->equivalentVariable(0),
         model->component("component2")->variable("variable2"));
 
-    EXPECT_EQ("b4da62", annotator->assignId(libcellml::Annotator::Type::CONNECTION, connection2));
+    EXPECT_EQ("b4da62", annotator->assignId(connection2));
```

Comment:
I don't think we should have a default for variable pair stuff (connections and mappings).  And if we did, it shouldn't be the connection as we never actually interact with a connection directly.  But my strong preference would be there is no default at all.

Reply:
Why would you not have a default?

Reply:
A default is useful for real CellML entities, where the parent item (reset) is the default, and owns something else (the resetVale).  Here, there is no CellML entity, there is no ownership, and the thing which people interact with (the explicit mapping of **variables** to each other, not the implicit connecting of **connections** to each other) isn't the default.  If you insist on having one of them, as a default, it should be the mapping, not the connection.  But even that is a stretch for me.

Reply:
Will change so that we have a default but to the `map_variables` instead of the connection.